doc/ref_arch/openstack/chapters/chapter01.md

```diff
@@ -55,6 +56,37 @@ OpenStack considers the following Four Opens essential for success:
 
 This OpenStack Reference Architecture is organised around the three major NFVI resource types as core services of compute, storage and networking, and a set of shared services of identity management, image management, graphical user interface, orchestration engine, etc. 
 
+<a name="1.4.1"></a>
+### 1.4.1 Exceptions
+
+CNTT specifies certain policies and [principles](https://github.com/cntt-n/CNTT/blob/master/doc/common/chapter00.md#2.0) but technology, performance and other considerations makes conformance to these primciples and policies impractical. CNTT specifies how to handle [non-conforming technologies](https://github.com/cntt-n/CNTT/blob/master/doc/common/policies.md#cntt-policies-for-managing-non-conforming-technologies). In general, non-coformance with policies is handled through a set of exceptions (please also see [Exception Types](https://github.com/cntt-n/CNTT/blob/master/doc/gov/chapters/chapter09.md#942-exception-types)). 
```

Comment:
Typo "primcipes"

Also the main thing that jumps out about this sentence is why does CNTT specify the policies if "technology, performance and other considerations makes conformance impractical"...    are saying in the future they may be practical or?

Reply:
@iangardner22 Can you review the new language? Thanks

Reply:
I feel we are slightly contradicting ourselves...  how about "CNTT strives to bring the industry towards a standard approach for NFVi but some technology options available today or specific carrier constraints mean that this is not always achievable therefore an exception process exists to manage this situation."

Reply:
might need a comma or two in my sentence sorry!

Reply:
@iangardner22 I had tried to leave the original content from RM the same but have now made changes in line with your suggestions -- please review. Thanks